src/entry_native.zig

```diff
@@ -10,163 +10,6 @@ const c = @import("c.zig").c;
 const Engine = @import("Engine.zig");
 const Options = Engine.Options;
 
```

Comment:
Seems like we could maybe put all native code into `native.zig`, instead of having `entry_native.zig` too? not sure what the semantic difference is intended to be

Reply:
I don't really like entry_native.zig, so sounds like a good plan to me. The whole organization is still a bit experimental I would say